client/cart/blocks/index.js

```diff
@@ -2,26 +2,10 @@
  * Internal dependencies
  */
 import { renderBNPLCartMessaging } from './product-details';
-import { getUPEConfig } from 'wcpay/utils/checkout';
 
 const { registerPlugin } = window.wp.plugins;
 
-const paymentMethods = getUPEConfig( 'paymentMethodsConfig' ) || {};
```

Comment:
I think using payment methods config as a filter is redundant here because we have enough flags in `client/cart/blocks/product-details.js` checking whether to show/initialize the PMME based on other (similar) limits/country logic coming from the PMME class on the back-end. I think this needs to be refactored and I've raised that in p1740588909669789-slack-CU6SYV31A, so that this can be addressed at the right time given multiple refactors currently ongoing